perfmetrics/scripts/ml_tests/README-usage.md

```diff
@@ -1,4 +1,4 @@
-# Commands to run animal_image_recognition_model
+# Commands to run fashion_items_image_recognition_model
```

Comment:
We can delete this file too, because it only talks about image_recognition model.